doc/user/sql/tail.md

```diff
@@ -36,6 +37,15 @@ Field | Represents
 `diff value` | Whether the record is an insert (`1`), delete (`-1`), or update (delete for old value, followed by insert of new value).
 `logical timestamp` | Materialize's internal logical timestamp.
 
+### AS OF
+
+If a timestamp is explicitly requested via `AS OF`, that timestamp will be used to determine what events to report rather than one automatically determined by Materialize.
+The timestamp provided to `AS OF` can be a number or a `TIMESTAMP` or `TIMESTAMPTZ` value.
```

Comment:
Probably should mention that numbers are treated as the number of seconds since the Unix epoch?